ClassLibrary1/Utilities/MenuUtilities.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Linq;
+
+namespace WhatShouldITest.Utilities
+{
+    public static class MenuUtilities
+    {
+        public static string GetDailySpecial(DayOfWeek dayOfWeek)
+        {
+            var _dailySpecials = new[]{
+                new {DayOfWeek = DayOfWeek.Monday,
+                    Special = "Half price tacos!" },
+                new {DayOfWeek = DayOfWeek.Tuesday,
+                    Special = "50 cent wings"},
+                new {DayOfWeek = DayOfWeek.Wednesday,
+                    Special = "2-for-1 Desserts"},
+                new {DayOfWeek = DayOfWeek.Thursday,
+                    Special = "5$ Burgers"},
+                new {DayOfWeek = DayOfWeek.Friday,
+                    Special = "All you can eat ribs" },
+                new {DayOfWeek = DayOfWeek.Saturday,
+                    Special = "Special Brunch" },
+                new {DayOfWeek = DayOfWeek.Sunday,
+                    Special = "Sunday, funday"}
+            };
+
+            return _dailySpecials.Where(s => s.DayOfWeek == dayOfWeek)
+                                 .Select(s => s.Special)
+                                 .First();
```

Comment:
Here it is. In the `GetDailySpecial` method in the `MenuUtilities` class. specifically the `s.DayOfWeek == dayOfWeek` portion of the code. That is the expression that will return either true or false, and therefore where the flow of the code can branch.

This is the logic that should be tested. Where the predicate is.